docs/en/observability/logs-stream.asciidoc

```diff
@@ -117,6 +99,17 @@ The following example shows the search results for logs with an `apm.error` data
 image::images/stream-logs-example.png[example search query on the logs stream page in {kib}]
 --
 
+[discrete]
+[[logs-stream-troubleshooting]]
+= Troubleshooting 
+
+If you're not seeing your log files in {kib}, check the following in the `elastic-agent.yml` file:
+
+NOTE: After installing the {agent}, you need to update the `elastic-agent.yml` in the directory described in the {fleet-guide}/installation-layout.html[installation layout].
+
+- Verify that the path to your logs file under `paths` is correct.
+- Verify that your API key is in <id>:<key> format. If not, your API key may be in an unsupported format, and you'll need to create an API key in *Beats* format. 
```

Comment:
There are commands like `elastic-agent status` that we could recommend here for debugging. @belimawr Any recommendations or links we should point to?

Reply:
The first things that come to mind are:
- [Troubleshooting guide](https://www.elastic.co/guide/en/fleet/current/fleet-troubleshooting.html)